Formula/kfctl.rb

```diff
@@ -0,0 +1,20 @@
+class Kfctl < Formula
+  desc "Kubeflow CLI tool"
+  homepage "https://www.kubeflow.org"
+  url "https://github.com/kubeflow/kubeflow/releases/download/v0.5.1/kfctl_v0.5.1_darwin.tar.gz"
```

Comment:
Don't use architecture specific sources. Preferably this shouldn't have any reference to macOS.

Reply:
Right now there is only a binary available from the project. [Actually two now](https://github.com/kubeflow/kubeflow/releases).

Reply:
https://docs.brew.sh/Acceptable-Formulae#we-dont-like-binary-formulae